frontend/website/docs/getting-started.md

```diff
@@ -52,7 +63,7 @@ If you're running a Coda node on a home or office machine, you'll have to set up
 
 Run the following commands to use MiniUPnP to reconfigure ports on your IP address:
 
-1. First run `ifconfig` to get your external IP address - you can find this in the output corresponding to the field `en0`:
+1. First run `ifconfig` to get your internal IP address - you can find this in the output corresponding to the field `en0` on macOS and `wlan0` on a linux system:
```

Comment:
The interface name isn't consistent on Linux. The identifier is generated I think pseudorandomly, and you may be on Ethernet or wireless. On my work laptop it's named `wlp59s0`.